tests/data/setup.py

```diff
@@ -14,6 +14,9 @@
 requirements = [
     'click>=5.0.0',
     'mock>=1.3.0',
+    'invenio-db[versioning]>=1',
+    'invenio-db[versioning,mysql]>=1',
```

Comment:
@david-caro IMHO we don't have to solve this limitation here since `pip-compile` already solves it for us.